wiki/Guides/Searching_and_Downloading_Beatmaps/fr.md

```diff
@@ -0,0 +1,3 @@
+# Comment chercher et télécharger des beatmaps
+
+Cet article est pour le moment uniquement disponible en [Chinois](/wiki/Guides/Searching_and_Downloading_Beatmaps?locale=zh).
```

Comment:
the link works, but is not the one we use.

```suggestion
Cet article est pour le moment uniquement disponible en [Chinois](/wiki/zh/Guides/Searching_and_Downloading_Beatmaps).
```

Reply:
this link was from en.md

Reply:
cc @cl8n

Reply:
it will fail the wikilink check, but I'm not sure if we want to leave this as-is. i'll wait for another opinion.

Reply:
I'd say the original language of a guide doesn't matter (same goes for other pages)

Reply:
you mean we should delete them?

Reply:
yes

Reply:
seems ticclick's comments were a misunderstanding, talked abt it on discord the other day